src/asmcnc/apps/systemTools_app/screens/screen_final_test.py

```diff
@@ -279,8 +279,7 @@ def set_x_steps(self):
         try:
             x_overstep = float(self.x_over_count.text)*self.x_calibration_scale_factor
             print(x_overstep)
-            self.m.send_any_gcode_command("$100 = " + str(float(self.m.s.setting_100) - x_overstep))
-            self.m.send_any_gcode_command("$$")
+            self.m.write_dollar_setting(100, float(self.m.s.setting_100) - x_overstep)
```

Comment:
Test these changes on the rig, just in case the reset after the stream causes any issues

Reply:
Just to check, what kind of issues should I be looking for with the reset? It looks like the values are updating correctly on the rig so I assume that means there's no issue?

Reply:
Yep just checking for any mis-setting or weirdness :).